src/query/api/v1/middleware/metrics.go

```diff
@@ -59,8 +74,15 @@ func ResponseMetrics(iOpts instrument.Options, route *mux.Route) mux.MiddlewareF
 				path = "unknown"
 			}
 
-			metrics := newRouteMetrics(iOpts)
```

Comment:
It seems like this was always created per request; moved it outside of the handler definition to line 58, could I get a double check that this is alright?

Reply:
the move looks correct, cc @ryanhall07